ISPP-G5/NexONG_Frontend
# Hotfix/Delete volunteer attendance

The 'attendance' screen for a volunteer has been removed. 

The possibility to stop being part of an event for volunteers has been made via a button that appears on the screens of the events you are part of.

To test it:
1. log in as a volunteer and you will be taken to the agenda screen.
2. click on an event you are not part of and you will see that the option does not appear.
3. click on an event you are part of and the option will appear, if you do this you will see that you will no longer be part of the event.
